pallets/dapp-staking-v3/src/lib.rs

```diff
@@ -186,6 +187,10 @@ pub mod pallet {
         #[pallet::constant]
         type NumberOfTiers: Get<u32>;
 
+        /// Tier ranking enabled.
+        #[pallet::constant]
+        type RankingEnabled: Get<bool>;
```

Comment:
Why not make number of ranks configurable instead, and let this be the number specifier?

Keeping it at 0 means legacy behavior, and [1,..., 15] new system is used.

Reply:
just to keep it simple, 1-10 is easy to reason abot

Reply:
Fair enough!